qcfractal/storage_sockets/view.py

```diff
@@ -0,0 +1,122 @@
+import io
+import pathlib
+from typing import Any, Dict, Union
+
+import numpy as np
+import pandas as pd
+
+from qcelemental.util.serialization import serialize
+
+# TODO(mattwelborn): names are hard
+from ..interface.collections import HDF5View
+from .storage_utils import get_metadata_template
+
+
+class ViewHandler:
+    def __init__(self, path: Union[str, pathlib.Path]) -> None:
+        """
+        Parameters
+        ----------
+        path: Union[str, Path]
+            Directory containing dataset views
+        """
+        self._path = pathlib.Path(path)
+        if not self._path.is_dir():
+            raise ValueError(f"Path in ViewHandler must be a directory, got: {self._path}")
+
+    def view_path(self, collection_id: int) -> pathlib.Path:
+        """
+        Get path of a view
```

Comment:
Small comment: we usually try to have these as a complete sentence with a period.